### PR TITLE
feat: index last device deployment status

### PR DESCRIPTION
**Pull request by kjaskiewiczz**
Ticket: [MEN-6422](https://northerntech.atlassian.net/browse/MEN-6422)

[MEN-6422]: https://northerntech.atlassian.net/browse/MEN-6422?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline --pr deployments/855

**Comment by kjaskiewiczz** [comment]
@mender-test-bot start pipeline --pr deployments/855 --pr deployments-enterprise/811

File: app/indexer/jobs.go [hunk 1]
```diff
@@ -108,14 +116,26 @@ func (i *indexer) processJobDevices(
 				break
 			}
 		}
+		for _, d := range deploymentsDevices {
+			if d.DeviceID == deviceID {
+				deploymentsDevice = &d
+				break
+			}
+		}
```
Comment (merlin-northern) [inline app/indexer/jobs.go:124]: consider creating deplyomentDevicesHashMap?

**Review by merlin-northern — APPROVED** [review]
one picky comment. lgtm.

File: app/indexer/jobs.go (hunk 1 above)
Comment (kjaskiewiczz) [reply app/indexer/jobs.go:124]: I will consider that ;)

**Pull request merged**